workers/ohsome_quality_analyst/api/api.py

```diff
@@ -210,6 +224,7 @@ async def post_report(name: ReportEnum, parameters: ReportRequestModel):
 
 async def _fetch_report(
     name: str,
+    includeSvg: bool = False,
```

Comment:
Here I would rename the variable to adhere to naming conventions (snake case for variables).
```suggestion
    include_svg: bool = False,
```